index.js

```diff
@@ -5,35 +5,42 @@ function restartGame() {
 let counter = 0;
 const timer = document.querySelector("#timer");
 //increasing the counter
-function inter() {
+function counterIncrease() {
   counter++;
   console.log();
   timer.innerHTML = "<b>" + counter + "</b>";
 }
-var interval;
+
+let interval;
 function setTimer() {
-  interval = setInterval(inter, 1000);
+  interval = setInterval(counterIncrease, 1000);
 }
 
-// open model when click on start game button
+const box = document.getElementById("model");
+const prevent_box = document.getElementById("prevent-box");
 
+// alert message when use try to access card before choosing the level 
+prevent_box.onclick = ()=>{
+  alert(`Oops...! \nChoose your Game level by clicking the Start Game button !`);
+}
+// open model when click on start game button
 function openModel() {
   counter = 0;
   clearInterval(interval);
   timer.innerHTML = "<b>" + 0 + "</b>";
-  const box = document.getElementById("model");
-  const prevent_box = document.getElementById("prevent-box");
-  prevent_box.style.display = "block";
-  box.style.transition = "all 0.5s";
-  box.style.transform = "scale(1)";
+  box.classList.add("box-condition");
 }
-// close model
-function closeModel() {
-  const box = document.getElementById("model");
-  const prevent_box = document.getElementById("prevent-box");
-  prevent_box.style.display = "none";
-  box.style.transition = "all 0.5s";
-  box.style.transform = "scale(0)";
+// close model when choosing levels and clicking cross button
+function closeModel(level) {
+  if (level != 0) {
+    setTimer();
+    prevent_box.classList.add("prevent-condition-none");
+  }
+
+  else {
+    prevent_box.classList.remove("prevent-condition-none");
+ }
+  box.classList.remove("box-condition");

```

Comment:
I have merged a PR that implements the game level. It accepts levels using the prompt. Just connect the buttons  you have created with that code and remove merge conflicts